modules/ROOT/pages/addition/agent-installation/self-registered.adoc

```diff
@@ -145,30 +149,18 @@ image::agents.png[width=800]
 . Find self-registered agent in list.
  ** If the agent is not in the list then go back to where the agent is running and check the logs.
     It may be that the server address is configured incorrectly or the TLS certificates are not correctly specified. 
-
-. Approve agent if required.
-** An agent in standby mode shows up in the list of agents in NOM UI with `Unauthorized` status. 
-+
-image::agent-unauthorized.png[width=800]
-To enable the agent to continue its normal execution, the agent needs to be approved from the NOM UI as shown below:
-.. Click on `...` agent action icon and click `Approve Agent`:
```

Comment:
Are we removing documentation of approval by this change? Meaning, we recommend to always use mTLS for self-registering?

Reply:
Yes